queries.sql

```diff
@@ -9,3 +9,47 @@ SELECT * from animals WHERE neutered = true;
 SELECT * from animals WHERE name NOT  IN ('Gabumon');
 SELECT * from animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
 
+
+
+
+
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+SELECT * FROM animals;
+ROLLBACK;
+
```

Comment:
- [ ] Good job on this query, but please you should verify if the change was made after rollback.